modules/dop/services/appcertificate/app_certificate.go

```diff
@@ -250,10 +259,10 @@ func (c *AppCertificate) ListAllAppCertificates(params *apistructs.AppCertificat
 // PushConfigs 推送Certificate配置到配置管理
 func (c *AppCertificate) PushConfigs(certificatePushReq *apistructs.PushCertificateConfigsRequest) error {
 	var (
-		pushReq = apistructs.PipelineCmsUpdateConfigsRequest{
-			PipelineSource: apistructs.PipelineSourceDice,
+		pushReq = &cmspb.CmsNsConfigsUpdateRequest{
+			PipelineSource: apistructs.PipelineSourceDice.String(),
```

Comment:
does pb support enum definition?

Reply:
Yes, but it's very hard to use.
And pipelineSource cannot be enumerated for `cdp-` prefixed scenario.

Reply:
will us support later? then all of this should be comment as TODO

Reply:
No.

Reply:
`PipelineSource` is not an Enum, **CANNOT** be enumerated.